### PR TITLE
add slim, borderless, stripped

### PR DESCRIPTION
**Pull request by Jdstuart13**

### CODE REVIEW

**Comment by jtrussell** [comment]
This looks good, just a couple suggestions:

- I think a small typo wormed it's way into your subconscious: `stripped` should `striped` 😄 
- We don't have an immediate need for it, but how would you feel about adding a `relaxed` fit for tables to go along with the `slim` fit? The only reason being then it would mirror the options provided by `au-list`.

File: packages/ivh-base-style/dist/ivh-base-style.css [hunk 1]
```diff
@@ -1575,6 +1562,33 @@ body::-webkit-scrollbar-track {
...
+  border-top:none;
+}
+
+.au-table-borderless th:first-child,
+.au-table-borderless td:first-child,
+.au-table-borderless > *:first-child,
+.au-table-borderless th,
+.au-table-borderless td,
+.au-table-borderless > * {
+  border: unset !important;
```
Comment (jtrussell) [inline packages/ivh-base-style/dist/ivh-base-style.css]: Is the `!important` required here?

Comment (Jdstuart13) [reply packages/ivh-base-style/dist/ivh-base-style.css]: important isn't necessary if you include thead th

Comment (Jdstuart13) [reply packages/ivh-base-style/dist/ivh-base-style.css]: I dislike !important so I will remove it.

File: packages/ivh-base-style/demo.html [hunk 2]
```diff
@@ -594,30 +594,178 @@ <h2>Tables</h2>
           <p>For tabular data only!</p>
         </div>
 
-        <div class="demo">
-          <h2>A True Table</h2>
+
+      <div class="demo">
```
Comment (evsheffield) [inline packages/ivh-base-style/demo.html]: Just a little style nitpick but I believe this section is missing one level of indentation.

File: packages/ivh-base-style/lib/css/table.css [hunk 3]
```diff
@@ -129,3 +130,29 @@
...
+
+.au-table-borderless thead th,
+.au-table-borderless th,
+.au-table-borderless td,
+.au-table-borderless > * {
+  border: unset;
+}
+
+.au-table-striped tbody > tr:nth-child(odd){
+  background-color: #EFF0F1;
```
Comment (evsheffield) [inline packages/ivh-base-style/lib/css/table.css]: You might consider adding this color to the theme list or using an already existing color if there's a good fit (https://github.com/iVantage/ivh-base/blob/master/packages/ivh-base-style/lib/index.js) and including it as a variable instead. If we end up adding the color, this has the added bonus of making that color available to use in code when importing the base styles package.

**Pull request merged**